assets/tuned/stalld/Makefile

```diff
@@ -17,11 +20,18 @@ DOCDIR	:=	$(DATADIR)/doc
 MANDIR	:=	$(DATADIR)/man
 LICDIR	:=	$(DATADIR)/licenses
 
-all: src/stalld.o
-	$(CC) -o stalld	 $(LDFLAGS) src/stalld.o $(LIBS)
+.PHONY:	all tests
 
-static: src/stalld.o
-	$(CC) -o stalld-static $(LDFLAGS) --static src/stalld.o $(LIBS)
+all:	stalld
+
+stalld: $(OBJ)
+	$(CC) -o stalld	 $(LDFLAGS) $(OBJ) $(LIBS)
+
+static: $(OBJ)
+	$(CC) -o stalld-static $(LDFLAGS) --static $(OBJ) $(LIBS)
+
+tests:
+	make -C tests
```

Comment:
We don't have tests as part of this PR so `make tests` wouldn't actually work?

```
$ make tests
make -C tests
make[1]: *** tests: No such file or directory.  Stop.
make: *** [Makefile:34: tests] Error 2
```

Reply:
That's right, tests do not ship with NTO to keep only the bare minimum inside.  Testing of stalld is done in the upstream repo.

Reply:
Ah are you just merging in their artifacts?

Reply:
Yes.  The only change I did is to remove the "tests" from the Makefile.  I should've probably changed the three Dockerfile(s) to do "make stalld" instead of just "make".  Tests were not part of old versions of stalld, so I just adjusted the makefile.

Reply:
Ack, so we want to keep it the same for ease of patching in the future?

Reply:
We can, so you want me to adjust the three Dockerfiles?  The only reason I didn't do this in the past was adjustments in three places and the fact the target `stalld` was changing a lot.  But I guess this is the last name (starvation_monitor/starved/stalld...).